BUILD.gn

```diff
@@ -15,9 +15,13 @@ config("deno_config") {
 rust_executable("deno") {
   source_root = "src/main.rs"
   extern = [ "$rust_build:libc" ]
-  deps = [
-    ":libdeno",
-  ]
+  deps = [ ":libdeno" ]
+}
+
+rust_executable("deno_nosnapshot") {
```

Comment:
Can you add a deno_cc_nosnapshot too. At the moment we're using deno_cc target because rust support for flatbuffers is not complete.

Also, before the `deno_nosnapshot` and `deno_cc_nosnapshot` targets, please add some comment about their purpose. Example comment (please edit)
```
# This target is for fast incremental development. When modifying the javascript runtime this target will not
# go through the extra process of building a snapshot and instead load the bundle from disk.
```